src/main/java/com/ldtteam/structurize/client/gui/WindowReplaceBlock.java

```diff
@@ -70,12 +71,12 @@
     /**
      * List of all item stacks in the game.
      */
-    private final List<ItemStack> allItems = new ArrayList<>();
+    private final List<ItemStorage> allItems = new ArrayList<>();
```

Comment:
I don't see the reason why this needs ItemStorage?
(same below)

Reply:
I make this ItemStorage so that I can do a contains check for duplicates

Reply:
Bleh, k